ovirt-guest-agent-centos/prep.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+/bin/rm -f /usr/share/ovirt-guest-agent/LockActiveSession.py{c,o}
+/bin/rm -f /usr/share/ovirt-guest-agent/LogoutActiveUser.py{c,o}
+
+# Fake being the host - To show the Atomic Host release
+/bin/rm -f /etc/redhat-release
+/bin/ln -sf /host/etc/redhat-release /etc/redhat-release
+
```

Comment:
could this be bind mount from the host in the config.json file?  Otherwise it will keep the redhat-release of the build host